README.md

```diff
@@ -3,69 +3,73 @@
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
 # Knife CLC
-
-This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like datacenters, templates and groups.
+This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like data centers, templates and groups.
 
 ## Installation
+There are three ways to install the Chef Knife plugin on your CenturyLink Cloud platform.
 
-If you're using [ChefDK](https://downloads.chef.io/chef-dk/), simply install the Gem:
+* If you're using [ChefDK](https://downloads.chef.io/chef-dk/), simply install the Gem:
 
 ```bash
 $ chef gem install knife-clc
 ```
 
-If you're using Bundler, add this line to your application's Gemfile:
+* If you're using Bundler:
```

Comment:
I would also revert `*` symbols. It adds some whitespace which breaks visual structure a bit. 

Here's how it looked before:

![image](https://cloud.githubusercontent.com/assets/902788/12298120/7f45cdc8-ba21-11e5-9ad9-58d743eb6ae9.png)

Here's how it looks with new changes:

![image](https://cloud.githubusercontent.com/assets/902788/12298150/a4595bfc-ba21-11e5-8a74-6ada302cce90.png)

Notice how `Add this line to your application's Gemfile`, `And then, execute:` are tabulated from `If you are using Bundler`. IMO, we should probably stick with old style.